x-pack/plugins/security_solution/public/cases/pages/case_details.tsx

```diff
@@ -29,11 +29,13 @@ export const CaseDetailsPage = React.memo(() => {
   }>();
   const search = useGetUrlSearch(navTabs.case);
 
-  useEffect(() => {
-    if (userPermissions != null && !userPermissions.read) {
-      navigateToApp(CASES_APP_ID, { path: getCaseUrl(search) });
-    }
-  }, [navigateToApp, userPermissions, search]);
+  if (userPermissions != null && !userPermissions.read) {
```

Comment:
why did we remove the. `useEffect`here?

Reply:
Yes, we definitely want the `useEffect`.